app/views/submissions/show.html.slim

```diff
@@ -1,194 +1,196 @@
-                br
-                = link_to image_tag(safe_submission_avatar(ansub), alt: "next", width: '60px', height: '60px'), ansub
-              - else
-                | Last submission!
-
-        .col-md-3
-          .card
-            .card-body style="text-align: center; padding: 5px;"
-              - unless nsub.blank?
-                | Next Submission by #{nsub.user.username} (&rarr;)
-                br
-                = link_to image_tag(safe_submission_avatar(nsub), alt: "next", width: '60px', height: '60px'), nsub
-              - else
-                | Last submission!
-
-      - has_permissions, error = @submission.can_be_commented_on_by(current_user)
-
-      .card
-        - if has_permissions
-          .card-header Comment On Submission
-
-          .card-body
-            | By commenting, you agree to adhere to DAD's guidelines for comment etiquette as defined 
-            a href="/help" here.
-            br
-            br
-            = form_for [@submission, Comment.new] do |f|
-              = f.text_area :body, placeholder: "Write your comment. Do your best to provide meaningful feedback: a step above surface-level critique, simple praise, or a token acknowledgement of feedback.", onkeyup: "charcountupdate(this.value)", id: "comment-box"
-              span#charcount 0/2000
-              = f.submit "Post Comment"
-        - else
-          .card-header Comment On Submission
-
-          .card-body
-            = error
-
-      - @comments.each do |comment|
-        .card.comment-card id="#{comment.id}" class="#{"alert alert-danger" if comment.soft_deleted}" style="padding: 0px;"
-          .card-header
-            b = link_to comment.user.username, comment.user
-            = comment.created_at.strftime(" %H:%M, %b %-d, %Y") 
-            span.quote-id onclick=("quote(#{comment.id});" if has_permissions)
-              = " No. #{comment.id}"
-            - if comment.soft_deleted
-              b  - Soft Deleted by #{safe_username(comment.soft_deleted_by)} -
-            = link_to ' <span class="fa fa-trash"></span>'.html_safe, [@submission, comment], method: :delete, data: { confirm: 'Are you sure?' } if comment.user_id == current_user&.id
-            | &nbsp;
-            - if logged_in_as_moderator
-              = form_with url: "#{submission_path(@submission)}/comments/#{comment.id}/mod_action", html: { style: 'width: auto; display: inline-block;' } do |f|
-                = f.text_field :reason, placeholder: "Reason", required: true, style: "width: 200px; display: inline-block; margin-bottom: 0px;"
-                = f.button "#{comment.soft_deleted ? "Undo Soft Delete" : "Soft Delete" }", name: 'toggle_soft_delete', class: "btn btn-sm btn-primary", style: "margin-left: 5px; line-height: 1;"
-
-          .card-body
-            = simple_format(comment.link_form)
+- provide(:title, "#{@submission.user.username} - #{@submission.display_title}")
```

Comment:
i'm not in a place where i can easily compile things (haven't gotten around to setting up WSL2 in the hotel), can you give me a screenshot of how followed submissions bubbling to the top appear?